README.md

```diff
@@ -132,6 +132,22 @@ This generator creates an example home controller and view which fetches and dis
 Options include:
 * __[Not recommended for embedded apps]__ `with-cookie-authentication` - This flag generates an authenticated home_controller, where the authentication strategy relies on cookies. By default, this generator creates an unauthenticated home_controller and protected sample products_controller.
 
+### Scopes Storage Generator
+```sh
+$ rails generate shopify_app:scopes_storage
+```
+
+This generator does the following:
```

Comment:
If I have the right end of the stick, this isn't needed for non-embedded apps, since cookies will expire eventually and trigger a re-auth, right? If that's the case, should we mention that at all here?

Reply:
We could include it for non-embedded apps as well because technically this is an issue that plagues all apps. The current way for any app to request new scopes is to wait for cookie expiry or rotate the app secret. I'm just not sure if the HomeController is the right place to do this check?

Reply:
That's true. I imagine that for non-embedded apps this might fit better into `AuthenticatedController`?